spec/models/cat_spec.rb

```diff
@@ -1,5 +1,30 @@
 require 'rails_helper'
 
 RSpec.describe Cat, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it "should validate presence of name" do
+    cat = Cat.create age:5, enjoys:"Long walks", image:"https://static.onecms.io/wp-content/uploads/sites/47/2020/11/30/cat-stuck-in-tree-298823311-2000.jpg"
+    expect(cat.errors[:name]).to_not be_empty 
+  end
+
+  it "should validate presence of age" do
+    cat = Cat.create name:"Morgan", enjoys:"Long walks", image:"https://static.onecms.io/wp-content/uploads/sites/47/2020/11/30/cat-stuck-in-tree-298823311-2000.jpg"
+    expect(cat.errors[:age]).to_not be_empty 
+  end
+
+  it "should validate presence of enjoys" do
+    cat = Cat.create name:"Morgan", age: 5, image:"https://static.onecms.io/wp-content/uploads/sites/47/2020/11/30/cat-stuck-in-tree-298823311-2000.jpg"
+    expect(cat.errors[:enjoys]).to_not be_empty 
+  end
+
+  it "should validate presence of image" do
+    cat = Cat.create name:"Morgan", age: 5, enjoys:"Long walks"
+    expect(cat.errors[:image]).to_not be_empty 
+  end
+
+  it "should validate presence of enjoys" do
```

Comment:
This test name is for the other enjoys test. This one should be something like 
`it "should validate minimum length of enjoys"`

Reply:
Updated...